WordPress/Tests/PHP/IniSetUnitTest.inc

```diff
@@ -0,0 +1,37 @@
+<?php
+
+ini_set('auto_detect_line_endings', true); // Ok.
```

Comment:
Might be good to see at least one entry with some unusual whitespace around and inside the parentheses, just to show it's not affecting the result.